controllers/index.js

```diff
@@ -44,8 +44,41 @@ exports.index = (req, res) => {
 
                 prs.push(returnedEvent);
             });
+
+            const requests = [];
+            _.forEach(prs, pr => {
+                const [repoOwner, repoName] = pr.repo_name.split('/');
```

Comment:
This errors for me locally, possibly due to the version of node I'm on, but I think we should support form node 6 and up. Are you able to change this?

Reply:
I thought destructuring was supported by version 6? See https://stackoverflow.com/questions/17379277/destructuring-in-node-js

I've no problem changing it though if needed